internal/cli/pipeline_inspect.go

```diff
@@ -127,6 +227,19 @@ func (c *PipelineInspectCommand) Flags() *flag.Sets {
 			Default: false,
 			Usage:   "Output the Pipeline as json.",
 		})
+
+		f.StringVar(&flag.StringVar{
+			Name:    "name",
+			Target:  &c.flagPipelineName,
+			Default: "",
+			Usage:   "Inspect a pipeline by name.",
+		})
+
+		f.IntVar(&flag.IntVar{
+			Name:   "run",
+			Target: &c.flagRunSequence,
+			Usage:  "Inspect a specific run sequence.",
+		})
```

Comment:
We should update the `Help()` below here in the description to talk about how asking for a run sequence will show the specific pipeline run

Reply:
Addressed, but Github didn't pick it up.